platform/api/oci/networking.go

```diff
@@ -15,194 +15,259 @@
 package oci
 
 import (
+	"context"
 	"fmt"
 
-	"github.com/oracle/bmcs-go-sdk"
+	"github.com/oracle/oci-go-sdk/core"
+	"github.com/oracle/oci-go-sdk/identity"
 )
 
-func (a *API) GetVCN(name string) (*baremetal.VirtualNetwork, error) {
-	vcns, err := a.client.ListVirtualNetworks(a.opts.CompartmentID, nil)
+const (
+	TCP  = "6"
```

Comment:
Do these need to be public? (Normally I'd agree they should be ALL_CAPS, but golang has other plans...(

Reply:
They dont, seems old habits die hard. I'll switch to private.